devolving_music/lib/elo_scoring.py

```diff
@@ -8,6 +8,10 @@ def probability(rating1, rating2):
 # k is a constant that determines how much you can gain or lose in one match .
 # d is 1 when player A wins 0 when they loseSS
 def elo_rating(ra, rb, k, d):
+	if(ra==None):
+		ra=0
```

Comment:
```suggestion
	if ra is None:
		ra = 0
```

- Unnecessary parentheses
- Spacing
- Always use `is None`, not `== None` ([explanation](https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none))

Reply:
![image](https://user-images.githubusercontent.com/22030302/175363778-05dc8e7b-db03-48e1-8bc1-a8319e24a822.png)
 It really doesn't matter